kmeans-sklearn/README.md

```diff
@@ -0,0 +1,43 @@
+# KMeans via sklearn (Python2)
+
+Two clustering examples using [sklearn](https://scikit-learn.org/stable/)
+
+It is assumed you already have a Binaris account. If you don't have an account yet, worry not, signing up is painless and takes just 2 minutes. Visit [Getting Started](https://dev.binaris.com/tutorials/python/getting-started/) to find concise instructions for the process.
+
```

Comment:
I think we definitely need a paragraph describing what we want to do: we will use kmeans to compress an imgae...